seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -424,9 +424,26 @@ def test_validate_anvil_vcf(self, mock_subprocess, mock_file_logger, mock_utils_
 
     @mock.patch('seqr.utils.file_utils.logger')
     @mock.patch('seqr.utils.file_utils.subprocess.Popen')
-    def test_get_anvil_vcf_list(self, mock_subprocess, mock_file_logger, mock_utils_logger):
+    def test_get_anvil_igv_options(self, *args):
+        # Requesting to load data from a workspace without an existing project
```

Comment:
```suggestion
```

Reply:
this comment is in this file twice, does it apply in both spots?

Reply:
nope good catch!